x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/advanced_view/detector_list.tsx

```diff
@@ -98,14 +101,16 @@ export const DetectorList: FC<Props> = ({ isActive, onEditJob, onDeleteJob }) =>
 
       <EuiFlexGrid columns={3}>
         {detectors.map((d, i) => (
-          <EuiFlexItem key={i}>
+          <EuiFlexItem key={i} data-test-subj={`mlAdvancedDetector ${i}`}>
             <EuiPanel paddingSize="m">
               <EuiFlexGroup>
                 <EuiFlexItem>
                   {d.detector_description !== undefined ? (
-                    <div style={{ fontWeight: 'bold' }}>{d.detector_description}</div>
+                    <div style={{ fontWeight: 'bold' }} data-test-subj="detectorDescription">
```

Comment:
Should this subject be `mlDetectorDescription`?

Reply:
It was originally planned to be nested subject only as it's not necessarily unique on its own, i.e. it's currently used as
```
`mlAdvancedDetector ${detectorIndex} > detectorIdentifier`
```
but it makes sense to keep our `ml` namespace here as well.

Reply:
Renamed in dfe2e17